Cargo.toml

```diff
@@ -90,9 +90,9 @@ opt-level = "z"
 lto = true
 
 [patch.crates-io]
-# kitsune2_api = { path = "../kitsune2/crates/api" }
-# kitsune2_core = { path = "../kitsune2/crates/core" }
-# kitsune2 = { path = "../kitsune2/crates/kitsune2" }
+kitsune2_api = { path = "../kitsune2/crates/api" }
+kitsune2_core = { path = "../kitsune2/crates/core" }
+kitsune2 = { path = "../kitsune2/crates/kitsune2" }
```

Comment:
This is just for local development right? Was this accidentally committed?

Reply:
This is here until the dependency in the kitsune repo is merged and published.

Reply:
Done: https://github.com/holochain/holochain/pull/4788/commits/2ac5f06c0b3e1168d0ee0de170b7a2a29bf8aa22